pkg/arvo/app/landscape/index.html

```diff
@@ -4,7 +4,7 @@
     <title>OS1</title>
     <meta charset="utf-8" />
     <meta name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+      content="width=device-width, initial-scale=1, shrink-to-fit=no,user-scalable=0"/>
```

Comment:
I think people will have a problem with this. See https://stackoverflow.com/a/46254706

Reply:
Who pinch zooms in Landscape atm? I don't think you'll be pinch zooming anywhere.

Reply:
I have a client named Chas who would disagree. He experiences pretty much every accessibility problem known to man, and I just don't think disabling pinch-to-zoom is a good approach. I mean, you tha boss, but Chas has helped me to realize that these things *matter* to people. I don't know that it's going to be a problem tomorrow or even this year, but this is a global change.

Reply:
Fair enough, I'll test, I'll test...

Reply:
Personally I think we should disable pinch to zoom.

Reply:
Idk. I get the idea that [aged users](https://accessiblewebsiteservices.com/accessible-web-development-how-to-allow-pinching-to-zoom-text/), for example, may need to look at text 2x. I'm happy to resolve this direct issue (omnibox autofocusing) while leaving the door open here.